lib/filter-rules-loading.js

```diff
@@ -0,0 +1,103 @@
+const path = require('path');
+const yaml = require('js-yaml');
+const fs = require('fs');
+
+const logger = require('./log');
+const { Console } = require('console');
+
+const SUPPORTED_IAC_EXTENSIONS = ['tf', 'yaml', 'yml','tpl','json'];
+function nestedCopy(array) {
+  return JSON.parse(JSON.stringify(array));
+}
+
+function injectRulesAtRuntime(filters) {
+  if (process.env.ACCEPT_IAC) {
+    logger.info(
+      { accept: process.env.ACCEPT_IAC },
+      'Injecting Accept rules for IAC extensions - Possible values tf, yaml, yml, json, tpl',
+    );
+    const extensions = process.env.ACCEPT_IAC.replace(/\s/g, '')
+      .split(',')
+      .filter((extension) => SUPPORTED_IAC_EXTENSIONS.includes(extension));
+    if (extensions.length <= 0) {
+      logger.error(
+        { accept: extensions },
+        '[MISCONFIGURATION] None of the requested ACCEPT IAC file extensions is compatible',
+      );
+    } else {
+      let template = nestedCopy(
+        filters.private.filter(
+          (entry) =>
+            entry.method == 'GET' && entry.path.includes('requirements'),
+        ),
+      );
```

Comment:
I'm a little worried that this is going to wind up with some weird, Python-related edge cases, but I don't think the odds of that actually happening are high enough to hold this change back.

Reply:
I'm just copying an entry so I don't need to maintain SCM specific things. If there is something else I should copy from it's fine, I just picked one of the file that is less likely to go away any time soon.